modules/ROOT/pages/configure-policy-template-source.adoc

```diff
@@ -1,31 +1,35 @@
-= Configuring Template Source for Policies in Runtime
+= Configuring Template Download Location for Policies
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, CloudFront
 
 
-When an application with a tracked flow is deployed to Mule runtime engine (Mule), the API gateway of that Mule environment communicates with API Manager to request for the applied policies.
+[PA: Nahuel what is the relationship between policy and templates and how are they used. We need to set some context as to why these templates are needed. What are these templates and how are they used, why are they needed?]
```

Comment:
Policies in Mule 4 are composed by three files: The business logic, that resides in a Policy Template (a deployable JAR file that contains the policy implementation), the configuration part (a YAML configuration file, where the policy parameters and metadata are defined) and the template's pom.xml file , which among other things, defines the policy dependencies.

Reply:
Done! Thanks!